types/validator.go

```diff
@@ -94,13 +94,13 @@ func (vc validatorCodec) Compare(o1 interface{}, o2 interface{}) int {
 //--------------------------------------------------------------------------------
 // For testing...
 
-func RandValidator(randPower bool, minPower int64) (*Validator, *PrivValidator) {
+func RandValidator(randPower bool, minPower uint64) (*Validator, *PrivValidator) {
 	privVal := GenPrivValidator()
 	_, tempFilePath := Tempfile("priv_validator_")
 	privVal.SetFile(tempFilePath)
 	votePower := minPower
 	if randPower {
-		votePower += int64(RandUint32())
+		votePower += RandUint64()
```

Comment:
Doesn't this have a (tiny) chance of overflowing.  Like (minPower / 2^64). Still not quite 0....

`addOrMax(votePower, RandUint64())`